frontend/src/app/testResults/uploads/CsvSchemaDocumentation.tsx

```diff
@@ -0,0 +1,296 @@
+              <h2 className="margin-top-0 ">
+                CSV schema documentation{" "}
+                <span className="text-secondary bg-white border-1px border-secondary font-body-3xs padding-x-1 padding-y-05 text-secondary margin-left-2 text-ttbottom">
+                  Pilot program{" "}
+                </span>
+              </h2>
+              <p className="usa-intro text-base">
+                How to format data for submission to SimpleReport via CSV
+                upload.
+              </p>
+              <p>
+                The SimpleReport standard CSV schema is a blend of the
+                Department of Health and Human Science's (HHS){" "}
+                <a
+                  href="https://www.hhs.gov/coronavirus/testing/covid-19-diagnostic-data-reporting/index.html"
+                  target="_blank"
+                  rel="noreferrer noopener"
+                  className="usa-link"
+                >
+                  requirements for COVID-19 test data
+                </a>{" "}
+                as well as those of numerous jurisdictions. This standard schema
+                will be accepted by state, tribal, local, or territorial (STLT)
+                health departments{" "}
+                <a
+                  href="/getting-started/organizations-and-testing-facilities/where-does-simplereport-work"
+                  className="usa-link"
+                >
+                  partnered
+                </a>{" "}
+                with SimpleReport.{" "}
+              </p>
+
+              <div className="usa-alert usa-alert--info margin-y-6">
+                <div className="usa-alert__body">
+                  <h3 className="usa-alert__heading font-body-md margin-top-05">
+                    About CSV upload
+                  </h3>
+                  This documentation will help you prepare a file for CSV
+                  upload. This feature is currently being piloted in select
+                  jurisdictions with organizations or facilities that have
+                  existing Electronic Medical Record (EMR) systems. Pilot
+                  partners are selected by recommendation from jurisdictions.
+                  Find out if your jurisdiction is{" "}
+                  <a
+                    href="/getting-started/organizations-and-testing-facilities/where-does-simplereport-work"
+                    className="usa-link"
+                  >
+                    partnered
+                  </a>{" "}
+                  with SimpleReport and{" "}
+                  <a
+                    href={
+                      "mailto:" +
+                      "support@simplereport.gov" +
+                      "?subject=Getting started with SimpleReport"
+                    }
+                    className="usa-link"
+                  >
+                    contact us
+                  </a>{" "}
+                  to learn more.
+                </div>
+              </div>
```

Comment:
Similar to the other page, recommend moving the blurb about the schema to the top of the next section and removing the "About CSV upload."